app-devel/scons/autobuild/defines

```diff
@@ -1,8 +1,9 @@
 PKGNAME=scons
 PKGSEC=devel
-PKGDEP="python-3 pypsutil"
-BUILDDEP="setuptools wheel"
 PKGDES="A Python replacement for Makefile-based build systems"
+PKGDEP="python-3 pypsutil"
+
```

Comment:
That is quite the weird formatting, but fine.

Reply:
was going to add some comment about builddep, but then found that's no longer necessary